README.md

```diff
@@ -24,61 +24,91 @@ Pip will handle all dependencies automatically and you will always install the l
+
+Using the Qiskit-provided `qiskit.primitives.Sampler` and `qiskit.primitives.Estimator` will not take you very far. The power of quantum computing cannot be simulated 
+on classical computers and you need to use real quantum hardware to scale to larger quantum circuits. However, running a quantum 
+circuit on hardware requires rewriting them to the basis gates and connectivity of the quantum hardware.
+The tool that does this is the [transpiler](https://qiskit.org/documentation/apidoc/transpiler.html) 
+and Qiskit includes transpiler passes for synthesis, optimization, mapping, and scheduling. However, it also includes a
+default compiler which works very well in most examples. The following code will map the example circuit to the `basis_gates = ['cz', 'sx', 'rz']` and a linear chain of qubits $0 \rightarrow 1 \rightarrow 2$ with the `coupling_map =[[0, 1], [1, 2]]`.
 
 ```python
-{'00': 513, '11': 511}
+from qiskit import transpile
+qc_transpiled = transpile(qc_example, basis_gates = ['cz', 'sx', 'rz'], coupling_map =[[0, 1], [1, 2]] , optimization_level=3)
 ```
 
-For further examples of using Qiskit you can look at the example scripts in **examples/python**. You can start with
-[using_qiskit_terra_level_0.py](examples/python/using_qiskit_terra_level_0.py) and working up in the levels. Also
-you can refer to the tutorials in the documentation here:
+For further examples of using Qiskit, look at the example scripts in **examples/python** and the tutorials 
+in the documentation here:
 
 https://qiskit.org/documentation/tutorials.html
 
 
-### Executing your code on a real quantum chip
+### Executing your code on real quantum hardware
+
+Qiskit provides an abstraction layer that lets users run quantum circuits on hardware from any vendor that provides a compatible interface. 
+The default way to use Qiskit is to have a runtime environment that provides optimal implementations of `sampler` and `estimator` for a given hardware. This runtime may involve using pre- and post-processing, such as optimized transpiler passes with error suppression, error mitigation, and, eventually, error correction built in. A runtime implements `qiskit.primitives.BaseSampler` and `qiskit.primitives.BaseEstimator` interfaces.
+
+* https://github.com/Qiskit/qiskit-ibm-runtime
 
-You can also use Qiskit to execute your code on a **real quantum processor**.
-Qiskit provides an abstraction layer that lets users run quantum circuits on hardware from any
-vendor that provides an interface to their systems through Qiskit. Using these ``providers`` you can run any Qiskit code against
-real quantum computers. Some examples of published provider packages for running on real hardware are:
+As Qiskit transitions to the runtime environment, some hardware is only supported with the ``providers`` interface; however, each provider may perform different types of pre- and post-processing and return outcomes that are vendor-defined:
```

Comment:
```suggestion
Qiskit also provides a lower level abstract interface for describing quantum backends. This interface, located in
in ``qiskit.providers``, defines an abstract `BackendV2` class that providers can implement to represent their
hardware or simulators to Qiskit. The backend class includes a common interface for executing circuits on the backends; however, in this interface each provider may perform different types of pre- and post-processing and return outcomes that are vendor-defined. Some examples of published provider packages that interface for will real hardware are:
```

Reply:
Two little tweaks, @mtreinish: in line 111, hyphenate `lower-level`, and note that the word "in" appears twice in a row (at the end of line 11 and beginning of line 12) :-)

Reply:
Thanks @abbycross I updated the suggestion.